### ISSUE TITLE
blocks led set color (scratch value color) do not work for "all"

### ISSUE THREAD

**Issue by mbonani** [Mobsya/scratch-vm#4]
### Expected Behavior

when the option all is selected all led should be the same, it is not the case 

### Actual Behavior

bottom color are false

but if you use the other option [top, bottom, bottom left, bottm right) the block is working

**Comment by mbonani** [comment]
Same problems for the block led change color it do not work for "all" and for" bottom "
it works individually

**Comment by mbonani** [comment]
There is a problem with callback function when they are imbricate

```
 this.sendAction('V_leds_top', rgb, () => {
                this._leds[0] = color;
                rgb.unshift(0);
                this.sendAction('V_leds_bottom', rgb, () => {
                    this._leds[1] = color;
                    rgb[0] = 1;
                    this.sendAction('V_leds_bottom', rgb, () => {
                        this._leds[2] = color;
                    });
                });
            });

```
 it make double the loops and network messages are also double or trippled.

**Comment by mbonani** [comment]
fix in commit 36f8ea15e54fbc3fd4ac44711801118804283935 and 05e0fb98fcf7eeaa84cea74462d2b97247340553
